driver/src/test/functional/com/mongodb/reactivestreams/client/gridfs/GridFSPublisherSpecification.groovy

```diff
@@ -378,43 +408,58 @@ class GridFSPublisherSpecification extends FunctionalSpecification {
     }
 
     def 'should cleanup when unsubscribing'() {
```

Comment:
This had to change due to request(1) being fixed and now we just set lots of data and ensure some is written and then later is cleaned up.